test/test_grid_dofhandler_vtk.jl

```diff
@@ -271,10 +271,11 @@ end
     hexgrid = generate_grid(Hexahedron,(2,2,1)) 
     topology = ExclusiveTopology(hexgrid)
     @test topology.edge_neighbor[1,11] == Ferrite.EntityNeighborhood(EdgeIndex(4,9))
+    @test getneighborhood(topology,hexgrid,EdgeIndex(1,11),true) == [EdgeIndex(4,9),EdgeIndex(2,12),EdgeIndex(3,10),EdgeIndex(1,11)]
     @test topology.edge_neighbor[2,12] == Ferrite.EntityNeighborhood(EdgeIndex(3,10))
+    @test getneighborhood(topology,hexgrid,EdgeIndex(2,12),true) == [EdgeIndex(3,10),EdgeIndex(1,11),EdgeIndex(4,9),EdgeIndex(2,12)]
```

Comment:
Can we relax these tests to not rely on the ordering of the elements? I.e. comparing sets instead of arrays?

Reply:
Ping @koehlerson

Reply:
should be done in a separate PR to improve the tests in general